kratos/processes/calculate_nodal_area_process.cpp

```diff
@@ -42,15 +42,15 @@ void CalculateNodalAreaProcess<THistorical>::Execute()
     }
 
     // We create the N and J0
-    Vector N;
-    Matrix J0;
+    struct tls_type
```

Comment:
This is with multiple variables in TLS. I followed PR #8195 for this. @philbucher

Reply:
perfect